lib/bn.d.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * These types are based on [1] from @ukstv and [2] from @MicahZoltu.
+ * If anything is wrong with these types, feel free to complain to @vsund.
+ * 
+ * [1] https://github.com/machinomy/types-bn
+ * [2] https://github.com/indutny/bn.js/pull/179
+ */
+
+
+export declare type Endianness = 'le' | 'be'
+
+export declare class BN {
+  static isBN(b: any): boolean;
+  static min(left: BN, right: BN): BN;
+  static max(left: BN, right: BN): BN;
+
+  constructor(number: number | string | number[] | Buffer, base?: number | 'hex', endian?: Endianness);
```

Comment:
Uint8Array is also a valid type for number, right?

Reply:
Not tried this out, but from looking at the code I'd say yes :)

Reply:
I am using Uint8Array inputs in a project with a bunch of tests and do not have any issues with it.